src/models/classes/customer.js

```diff
@@ -0,0 +1,35 @@
+ class Customer
+ {
+     /**
+      * @constructor Represents a Customer
+      *
+      * @param products
+      */
+    constructor(products)
+    {
+        this.name = this.generateName();
+        this.products = products;
```

Comment:
Perhaps have a customer with a field `order`, that contains `products` and `orderCost`. Products and orderCost do not seem like properties of a customer to me, whereas an order does. Any thoughts?